internal/custom_executable/yes/yes_test.go

```
@@ -0,0 +1,133 @@
+	select {
+	case <-ctx.Done():
+		// Context timed out, ensure the process is terminated
+		if cmd.Process != nil {
+			cmd.Process.Kill()
+		}
+	case <-readDone:
+		// Reading completed
+	}
+
+	// Wait but ignore error as we expect the process might be killed
+	cmd.Wait()
```

golangci-lint (1.64.8)

Error return value of `cmd.Wait` is not checked

(errcheck)